spec/fixtures/proofing/lexis_nexis/ddp/request.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
[My earlier comment](https://github.com/18F/identity-idp/pull/6684#discussion_r937943618) was made for the request fixture, not the response fixture (oops) 

But maybe would our specs be easier to maintain if we inlined this into the spec file instead of putting it in a fixture? JSON is easier to inspect than XML